bin/validators.js

```diff
@@ -48,6 +48,24 @@ module.exports = {
     return Boolean(firstOption && firstOption.allowIndentationTabs);
   },
 
+  "unicorn/template-indent"({ options }) {
+    if (!options[0]) {
+      return false;
+    }
+
+    const firstOption = options[0];
+
+    return !Boolean(
```

Comment:
Should we consider https://github.com/Sec-ant/prettier-plugin-embed here?

Reply:
Doesn't seem official, so maybe people can configure it themselves using the documentation we created

I'd prefer merging as is